packages/web-components/fast-components/src/anchor/index.ts

```diff
@@ -30,9 +30,9 @@ export class Anchor extends FoundationAnchor {
         oldValue: AnchorAppearance,
         newValue: AnchorAppearance
     ): void {
-        if (oldValue !== newValue) {
-            this.classList.add(newValue);
+        if (this.$fastController.isConnected) {
```

Comment:
Is this the case in Button as well?

Reply:
Looks like we are setting a class here, which likely isn't necessary. Can we add a note above to remove this in the next major version?